src/runtime/runtime.cs

```diff
@@ -1884,5 +1886,33 @@ internal static IntPtr PyMem_Realloc(IntPtr ptr, long size)
 
         [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
         internal static extern int Py_MakePendingCalls();
+
+        internal static void SetNoSiteFlag()
+        {
+            var loader = LibraryLoader.Get(OperatingSystem);
+
+            IntPtr dllLocal;
+            if (_PythonDll != "__Internal")
+            {
+                dllLocal = loader.Load(_PythonDll);
+            }
+
+            try
+            {
+                Py_NoSiteFlag = loader.GetFunction(dllLocal, "Py_NoSiteFlag");
+
+                if (Is32Bit)
+                    Marshal.WriteInt32(Py_NoSiteFlag, 1);
+                else
+                    Marshal.WriteInt64(Py_NoSiteFlag, 1);
```

Comment:
This is not correct, `int` on Windows is 32bit.

Reply:
I don't understand, the code specifies Int32 or Int64 depending on arch. What would you suggest this be instead?

Reply:
The arch defines the Pointer width, this flag is not a pointer but an `int`. It's actually 32bit pretty much everywhere, not only on Windows (the distinction has to be made for `long`). Drop the 64bit path and this is fine.

Reply:
Thanks, changes pushed.